CONTRIBUTING.md

```diff
@@ -74,6 +74,39 @@ accepting contributions.
    you like, especially if we can't incorporate your recommendations. But please also honor
    our [Governance Rules](GOVERNANCE.md).
 
+## Branching and Workflow
+
+We use a centralized approach that leaves `main` as the bleeding edge and has version branches for bug and feature
+fixes in the specific version. Since we use [Semantic Versioning](https://semver.org/) we need to provide ways that
+we can addequately maintain and/or add features to existing versions, without breaking the backwards compatibility of
+those versions.
+
+* The `main` branch is the source of truth for the bleeding edge of the project. It should be used for applying all new
+  features and bugfixes that affect new and existing features.
+* The version branch (e.g. `1.x`) is used for bugfixes specific to features in that version. Bugfixes that are applied
+  to `main` and also apply to the version branch should be first merged to `main` and then applied to the version
+  branch. Fixes that apply ONLY to the version branch should target that branch specifically.
+* New features should be developed against `main` **first**. If the new feature is applicable to a version branch and
```

Comment:
Why are new features dealt with differently than bugfix branches? Seems like they should both be dealt with the same.